acceptance-tests/pom.xml

```diff
@@ -0,0 +1,142 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.jenkins.plugins</groupId>
+    <artifactId>acceptance-tests</artifactId>
+    <name>Bitbucket Server Integration Plugin Acceptance Tests</name>
+    <description>Includes the integration/acceptance tests for the Bitbucket Server Jenkins integration plugin</description>
+    <packaging>jar</packaging>
+    <version>1.1.1-SNAPSHOT</version>
+
+    <properties>
+        <jenkins.version>2.176.1</jenkins.version>
```

Comment:
This should be defined at the parent level. As of now, it is too easy for it to go out of place with plugin Jenkins version

Reply:
Please see response to the previous comment.